recipes/htop/COPYING

```diff
@@ -0,0 +1,355 @@
+        GNU GENERAL PUBLIC LICENSE
```

Comment:
We shouldn't need to add one.

The feedstock will get a BSD 3-Clause license as it is the standard.

Adding the `license_file` portion in the recipe was the only thing we needed to do. However, it pulls the license file from the `htop`'s source. So it isn't using this file either.